extensions/interactions/PencilCodeEditor/PencilCodeEditor.html

```diff
@@ -38,3 +37,17 @@
 <script type="text/ng-template" id="shortResponse/PencilCodeEditor">
   <pre><[answerCode | truncateAtFirstLine]></pre>
 </script>
+
+<script type="text/ng-template" id="modals/confirmationModal">    
+  <div class="modal-header">
+    <button type="button" class="close" ng-click="close(false)" data-dismiss="modal" aria-hidden="true">&times;</button>
```

Comment:
Ok, should I change it to "PencilCodeEditorConfirmationModal" to make it more specific ?